src/utils/Api.js

```diff
@@ -0,0 +1,125 @@
+class Api {
+  constructor({ baseUrl, headers }) {
+    (this._baseUrl = baseUrl), (this._headers = headers);
+  }
+
+  getAppInfo() {
+    return Promise.all([this.getInitialCards(), this.getUserInfo()]);
+  }
+
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: this._headers,
+    })
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        } else {
+          Promise.reject(`Error ${res.status}`);
+        }
+      })
+      .catch(console.error);
+  }
+
+  loadCards() {
+    fetch(`${this._baseUrl}/cards/`, {
+      headers: this._headers,
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        data.forEach((card) => {
```

Comment:
Please, delete `loadCards` because it's not used, and the code is wrong inside it